### PR TITLE
add new plugins to Manifest and `/integrations/*`

### PR DESCRIPTION
**Pull request by teallarson**
- [X] Mailjet
- [X] Mixpanel
- [X] Calculated Properties (manifest only, not on integrations list)

Did not add individual integration pages for Mailjet or Mixpanel for this PR.

### CODE REVIEW

File: data/plugins/calculated-property.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  priority: 0,
+  slug: "calculated-property",
+  logo: "calculated-property.png",
+  primaryType: "source",
+  otherTypes: [],
+  category: "transform tool",
+  badge: "",
+  showLink: false,
+  packageName: "@grouparoo/calculated-property",
+};
```
Comment (tealjulia) [inline data/plugins/calculated-property.ts:16]: This file generates the manifest entry for calculated properties... it is not imported into the `/integrations` page at all but is still needed for the manifest.

File: data/plugins/index.ts [hunk 2]
```diff
@@ -95,7 +97,9 @@ export const PluginData: Plugin[] = [
   Intercom,
   Iterable,
   Mailchimp,
+  Mailjet,
   Marketo,
+  Mixpanel,
```
Comment (tealjulia) [inline data/plugins/index.ts:102]: Adds Mailjet and Mixpanel to the `/integrations` page

**Comment by bleonard** [comment]
Is it true that we should wait to merge this until the 0.6 when the plugins are released? I think that's right because UI config reads the manifest.

**Comment by teallarson** [comment]
> Is it true that we should wait to merge this until the 0.6 when the plugins are released? I think that's right because UI config reads the manifest.

@bleonard  Sounds right.  I just figured out why it was being erased from the manifest every time I pushed.  I'm thinking this commit should have it good to go!

**Review by andyjih — CHANGES_REQUESTED** [review]
Looks good! The only note is that the calculated property image looks a bit skewed. 

Screen Shot 2021-08-31 at 9 49 27 AM

**Comment by teallarson** [comment]
> Looks good! The only note is that the calculated property image looks a bit skewed.
> 
> Screen Shot 2021-08-31 at 9 49 27 AM

Thanks!  I think we actually don't want it to display calculated properties at all on this page.  I did make it square to display properly in UI config, but I also removed it from the `/integrations` page.

**Review by andyjih — APPROVED** [review]

**Pull request merged**